Scale_space_reconstruction_3/include/CGAL/Scale_space_reconstruction_3/Alpha_shape_mesher.h

```diff
@@ -181,7 +181,7 @@ class Alpha_shape_mesher
   Alpha_shape_mesher (FT squared_radius,
                       bool separate_shells = false,
                       bool force_manifold = false,
-                      FT border_angle = 45.)
+                      FT border_angle = CGAL_PI/4.0)
```

Comment:
is this a parameter exposed in the API at some point?

Reply:
Good point.  In fact @sgiraudot will give us further functions where we should unify.